vision_datasets/common/data_manifest.py

```diff
@@ -96,14 +98,30 @@ def __init__(self, id, img_path, width, height, labels):
                 classification: [c_id] for multiclass, [c_id1, c_id2, ...] for multilabel;
```

Comment:
still need a comment for matting for labels

Reply:
I am debating whether we should have a list or a single matting PIL here (I understand our current data has 1 matting for 1 image).... @shonohs thoughts?

Reply:
resolved. now the labels for image matting are in a list format